system/modules/core/classes/Backend.php

```diff
@@ -757,33 +757,27 @@ public static function addFileMetaInformationToRequest($strUuid, $strPtable, $in
 		switch ($strPtable)
 		{
 			case 'tl_article':
-				$objPage = $db->prepare("SELECT * FROM tl_page WHERE id=(SELECT pid FROM tl_article WHERE id=?)")
-							  ->execute($intPid);
+				$objPage = \PageModel::findOneBy(array('tl_page.id=(SELECT pid FROM tl_article WHERE id=?)'), array($intPid));
```

Comment:
https://github.com/contao/core/blob/master/system/modules/core/library/Contao/Model.php#L821

The table name must always be specific because of potential relation joins.